crates/astria-bridge-withdrawer/src/bridge_withdrawer/mod.rs

```diff
@@ -260,6 +281,46 @@ impl BridgeWithdrawer {
     }
 }
 
+async fn make_signer(
+    frost_threshold_signing_enabled: bool,
+    frost_min_signers: usize,
+    frost_public_key_package_path: String,
+    frost_participant_endpoints: Vec<String>,
+    sequencer_key_path: String,
+    sequencer_address_prefix: String,
+) -> eyre::Result<Arc<dyn Signer>> {
+    let signer: Arc<dyn Signer> = if frost_threshold_signing_enabled {
```

Comment:
Noted elsewhere - I think making this an enum (static dispatch) rather than a trait (dynamic dispatch) might end up being cleaner and easier to understand.